kodilla-collections/src/main/java/com/kodilla/collections/arrays/homework/CarsApplication.java

```diff
@@ -0,0 +1,37 @@
+package com.kodilla.collections.arrays.homework;
+
+import com.kodilla.collections.interfaces.homework.Car;
+import com.kodilla.collections.interfaces.homework.Subaru;
+import com.kodilla.collections.interfaces.homework.Tesla;
+import com.kodilla.collections.interfaces.homework.Volvo;
+
+import java.util.Random;
+
+public class CarsApplication {
+    public static void main(String[] args) {
+        Random random = new Random();
+        Car[] cars = new Car[random.nextInt(15) + 1];
+
+        for (int i = 0; i < cars.length; i++)
+            cars[i] = drawCar();
+
+        for (Car car : cars)
+            CarUtils.describeCar(car);
+    }
+
+    public static Car drawCar() {
+        Random random = new Random();
```

Comment:
nie ma sensu tworzyc obiektu random, przy kazdym wywolaniu metody. Lepiej aby random byla stala
private static Random random = new Random()

Reply:
jak stworzysz te zmienna klasowa, to w main random tez nie bedzie potrzebny